week1_KimChaeHyun/src/components/tab/FilterTab.module.css

```
@@ -0,0 +1,122 @@
+.tabContainer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0.5rem 1rem;
+}
+
+.tabs {
+  position: relative;
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+
+  width: 28rem;
```

```suggestion
  gap: 2rem;
```
gap 수정하고, width 값 없애고